.circleci/config.yml

```diff
@@ -1661,31 +1664,71 @@ jobs:
                   name: "Building package bundle"
                   working_directory: "./release"
                   command: npm run zx -- --experimental ci-steps/package-bundles.mjs --version=<< pipeline.parameters.graviteeio_version >>
-            - keeper/env-export:
-                  secret-url: keeper://Mqmplmfu17bDR5XRLmO1mQ/field/password
-                  var-name: AWS_ACCESS_KEY_ID
-            - keeper/env-export:
-                  secret-url: keeper://3-pU56sIqcyWWw7HxhxjaQ/field/password
-                  var-name: AWS_SECRET_ACCESS_KEY
-            - when:
-                  condition: << pipeline.parameters.dry_run>>
-                  steps:
-                      - aws-s3/sync:
-                            arguments: |
-                                --endpoint-url https://cellar-c2.services.clever-cloud.com \
-                                --acl public-read
-                            from: ./release/.tmp/<< pipeline.parameters.graviteeio_version >>/dist
-                            to: "s3://gravitee-dry-releases-downloads/graviteeio-apim"
+            - run:
+                  name: "Comparing bundle size with previous version"
+                  working_directory: "./release"
+                  command: |
+                    npm run zx -- --experimental ci-steps/check-bundle-size.mjs
+                    echo "export BUNDLE_SIZE_MESSAGE=$(cat .tmp/bundle-size.md)" >> $BASH_ENV
             - when:
-                  condition:
-                      not: << pipeline.parameters.dry_run>>
-                  steps:
-                      - aws-s3/sync:
-                            arguments: |
-                                --endpoint-url https://cellar-c2.services.clever-cloud.com \
-                                --acl public-read
-                            from: ./release/.tmp/<< pipeline.parameters.graviteeio_version >>/dist
-                            to: "s3://gravitee-releases-downloads/graviteeio-apim"
+                condition:
+                  not: << pipeline.parameters.dry_run>>
+                steps:
+                  - slack/notify:
+                      channel: C02NGT20S4W
```

Comment:
Don't know if we should notify this channel or `api-management_team_notifications`